k8s/namespaces/reform-scan/reform-scan-notification-service/aat.yaml

```diff
@@ -3,7 +3,6 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: reform-scan-notification-service
-  namespace: reform-scan
   annotations:
     flux.weave.works/automated: "true"
     flux.weave.works/tag.java: glob:prod-*
```

Comment:
Shouldn't be needed
```suggestion
 
```

Reply:
Updated all yaml files with the suggestions